package/com.unity.formats.usd/Runtime/Scripts/IO/Materials/StandardShaderExporter.cs

```diff
@@ -225,6 +223,35 @@ public class StandardShaderExporter : ShaderExporterBase {
         surface.diffuseColor.defaultValue = new Vector3(c.r, c.g, c.b);
       }
 
+      // Standard Shader has 4 modes:
+      // 0: Opaque - no opacity/transparency
+      // 1: Cutout - opacityThreshold should be used to cut off based on alpha values (in _MainTex.a)
+      // 2: Fade - opacity should be used, but no opacityThreshold
+      // 3: Transparent - opacity should be used, but no opacityThreshold.
+      // Note: not quite sure if and how the difference between Fade and Transparent should be handled in USD.
+      int shaderMode = 0;
+      if(mat.HasProperty("_Mode")) {
```

Comment:
Maybe define a local enum for this (if there isn't a public enum) to avoid magic numbers.

Reply:
It's internal and in the shader it's actually a magic number ;)
I made a local enum to reflect the internal one though.